build-scripts/validate_pr_in_jenkins

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if[ -z "$ghprbPullId" ]; then exit 0; fi
```

Comment:
I had to fix this line: needs to have a space after `if`, because it's the shell keyword:
`if [ -z "$ghprbPullId" ]; then exit 0; fi`